+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "1643"
    creation_ts: 2012-08-09 12:53:00 +0200
    short_desc: ft_redefinetrial produces error message when cutting data into snippets
    delta_ts: 2013-08-14 09:36:27 +0200
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: All
    op_sys: All
    bug_status: UNCONFIRMED
    resolution:
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P3
    bug_severity: normal
    target_milestone: '---'
    everconfirmed: "0"
    reporter:
      +content: s.vanpelt
      +@name: Stan van Pelt
    assigned_to:
      +content: s.vanpelt
      +@name: Stan van Pelt
    cc:
      - j.schoffelen
      - r.oostenveld
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "6977"
        comment_count: "0"
        who:
          +content: s.vanpelt
          +@name: Stan van Pelt
        bug_when: 2012-08-09 12:53:46 +0200
        thetext: |-
          I have a dataset that is an appending of 4 other dataset (output from ft_preprocessing), which I want to cut into non-overlapping snippets of 0.5s. If I do this, the latest version of ft_redefinetrial gives an error in some cases (some subjects, not all), on line 236: "some of the new trials need to be constructed from more than one input trial. This is not supported."
          However, I don't think this operation should produce an error in such cases.
      - +@isprivate: "0"
        commentid: "6978"
        comment_count: "1"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-08-09 16:51:58 +0200
        thetext: can you provide a small piece of data and a test script to reproduce the problem?
      - +@isprivate: "0"
        commentid: "11146"
        comment_count: "2"
        who:
          +content: s.vanpelt
          +@name: Stan van Pelt
        bug_when: 2013-08-14 09:32:03 +0200
        thetext: can't replicate the error any more
      - +@isprivate: "0"
        commentid: "11147"
        comment_count: "3"
        who:
          +content: j.schoffelen
          +@name: Jan-Mathijs Schoffelen
        bug_when: 2013-08-14 09:36:27 +0200
        thetext: That's good, then I suggest to close this one...
